pkg/process/net/windows_pipe.go

```diff
@@ -32,3 +60,17 @@ func (wp *WindowsPipeListener) GetListener() net.Listener {
 func (wp *WindowsPipeListener) Stop() {
 	wp.conn.Close()
 }
+
+// DialSystemProbe connects to the system-probe service endpoint
+func DialSystemProbe(_ string, _ string) (net.Conn, error) {
```

Comment:
nit: I expected this function in `common_windows.go` because the other OS versions were in `common_*.go`

Reply:
The original placeholder function was located in windows_pipe.go.  Since future iterations intend to add more specialized Windows named pipe logic, it makes sense to separate it from the generic common_windows.go. It is easy to bloat a "common" file over time.